src/Microsoft.Extensions.Logging.Testing/AssemblyTestLog.cs

```diff
@@ -228,7 +228,7 @@ private static string GetAssemblyBaseDirectory(string assemblyName, string baseD
         {
             if (!string.IsNullOrEmpty(baseDirectory))
             {
-                return Path.Combine(baseDirectory, assemblyName, RuntimeInformation.FrameworkDescription.TrimStart('.'));
+                return Path.Combine(baseDirectory, assemblyName, new DirectoryInfo(AppContext.BaseDirectory).Name);
```

Comment:
A bit hacky because this relies on the AppContext.BaseDirectory to be in bin/{Config}/{tfm} but I don't know if there's a better way to find the TFM that's compatible with 4.6.1. Since we launch tests via xunit this should work.

Reply:
Or we can hardcode a dictionary :) How many frameworks do we run on 3-4?

Reply:
Hardcoding a dictionary is not ideal since when we change the TFMs, like we did for netcoreapp2.2, we'd also need to update the dictionary. I'll look into the other suggestion though.

Reply:
It happens once a year. And fallback would be there.